README.md

```diff
@@ -7,7 +7,10 @@ With elements based on the popular game Among Us, AmongUps has two players fight
 
 ## Screenshots
 
-
+![Start Screen](./Media/StartScreen.jpg)
+![Controls](./Media/Controls.jpg)
+![Game Start](./Media/GameStart.jpg)
+![Power Up Screen](./Media/PowerUpScreen.jpg)
 
 ## Tech Stack
 
```

Comment:
Towards the end of this file, there is a typo in my name. it is Ajwad `Shaikh` and not `Shaik`
Might wanna change that too

Reply:
Fixed, sorry about that :(